base_import_async/models/base_import_async.py

```diff
@@ -30,7 +30,9 @@
 from openerp.models import fix_import_export_id_paths
 from openerp.tools.translate import _
 
-from openerp.addons.connector.queue.job import job, related_action
+from openerp.addons.connector.queue.job import (
```

Comment:
No I'm not happy with the `try` `except` hack. I can tolerate it for the python dependencies but in this case it's not applicable .... https://github.com/acsone/connector-interfaces/blob/dc8ce768bf0c63db18f08c1922af97870c9efb5d/base_import_async/models/base_import_async.py#L133
`job` is required by the code to decorate functions defined at the package level. Even if you protect the import, it's not possible to import this module without the connector.